engine/release-notes/index.md

```diff
@@ -15,6 +15,51 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.20
+2022-10-18
+
+This release of Docker Engine comes with mitigations against a Git vulnerability,
+tracked in [CVE-2022-39253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-39253){:target="_blank" rel="noopener"},
+and updated handling of `image:tag@digest` image references.
+
+The Git vulnerability allows a maliciously crafted Git repository, when used as a
+build context, to copy arbitrary file system paths into resulting containers/images,
+this vulnerability could affect clients (when using the classic builder) or the
+happen on the client.
+
+Users are also recommended to update their version of Git to prevent the vulnerability.
+Details can be found in the ["Git security vulnerabilities announced"](https://github.blog/2022-10-18-git-security-vulnerabilities-announced/){:target="_blank" rel="noopener"}
+GitHub blogpost.
+
+
+### Client
+
+- Added a mitigation against [CVE-2022-39253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-39253){:target="_blank" rel="noopener"},
+  when using the classic Builder and using a Git repository as build context.
+
+### Daemon
+
+- Updated handling of `image:tag@digest` references. When pulling an image using
+  the `image:tag@digest` ("pull by digest"), image resolution happens through
+  the content-addressable digest and the `image` and `tag` are not used. While
+  this is expected, this could lead to confusing, and could potentially be
+  exploited through social engineering to run an image that is already present
+  in the local image store. Docker now checks if the digest matches the repository
+  (`image`) used to pull the image, and otherwise will produce an error.
+
+
+### Builder
+
+- Updated handling of `image:tag@digest` references. Refer to the "Daemon" section
+  above for details.
+- Updated the classic Builder and updated BuildKit to [v0.8.3-31-gc0149372](https://github.com/moby/buildkit/commit/c014937225cba29cfb1d5161fd134316c0e9bdaa){:target="_blank" rel="noopener"},
+  provide a mitigation against [CVE-2022-39253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-39253){:target="_blank" rel="noopener"}
+  when using a Git repository as build context with the classic builder.
```

Comment:
```suggestion
- Added a mitigation to the classic Builder and updated BuildKit to [v0.8.3-31-gc0149372](https://github.com/moby/buildkit/commit/c014937225cba29cfb1d5161fd134316c0e9bdaa){:target="_blank" rel="noopener"},
  for [CVE-2022-39253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-39253){:target="_blank" rel="noopener"}
  when using the classic Builder with a Git URL as the build context.
```

Reply:
I dropped the "when using the classic Builder " here, as in this case it's the daemon-side, so could be for _both_